cob_scan_unifier/src/scan_unifier_node.cpp

```diff
@@ -26,6 +26,10 @@ ScanUnifierNode::ScanUnifierNode()
   nh_ = ros::NodeHandle();
   pnh_ = ros::NodeHandle("~");
 
+  // Init TF-Buffer
+  p_tfBuffer = new tf2_ros::Buffer();
+  p_tfListener = new tf2_ros::TransformListener(*p_tfBuffer, true);
```

Comment:
but this node does not use `tf2`...so what kind of  errors are you experience?
(CI fails as this node does not have those members you are initializing)

Reply:
I do not get unified scans. As they are not transformed since tf is not initalized... would it be solution to move to tf2 or should I implement as in ipa320/cob_control#202 ?